make/deps.mk

```diff
@@ -53,62 +44,130 @@ ifeq ($(USER),travis)
+dep_folsom = git https://github.com/boundary/folsom 0.8.2
+# used by hangups
 
-dep_couchbeam = git https://github.com/2600hz/couchbeam 2600hz
-###dep_couchbeam = git https://github.com/benoitc/couchbeam 1.4.1
-### waiting for pull requests
-### https://github.com/benoitc/couchbeam/pull/158
-### https://github.com/benoitc/couchbeam/pull/164
-### https://github.com/benoitc/couchbeam/pull/165
-### https://github.com/benoitc/couchbeam/pull/166
-### https://github.com/benoitc/couchbeam/pull/174
+## Code reloaders for dev VMs, uncomment if desired
+# dep_fs_event = git https://github.com/jamhed/fs_event 783400da08c2b55c295dbec81df0d926960c0346
+# dep_fs_sync = git https://github.com/jamhed/fs_sync 2cf85cf5861221128f020c453604d509fd37cd53
+
+dep_gen_smtp = hex 0.13.0
+# used by teletype, notify, and fax
+
+dep_getopt = hex 1.0.1
+# used in some scripts/ and sup
+
+dep_gproc = hex 0.8.0
+# used by kazoo_events, webseq, konami, acdc, ecallmgr, callflow
+
+dep_hep = git https://github.com/2600hz/hep-erlang a8f62dbc756fd3d4be1e2f0a6cec47a23f6cd18a
+# Homer encapsulation protocol
+# merged lazedo/hep changes
+
+# dep_horse = git https://github.com/ninenines/horse 4dc81d47c3116b38af673481402f34ce03f8936b
+# used by kazoo_stdlib in some test modules
+# uncomment if you want to do simple perf testing of code
+
+dep_inet_cidr = git https://github.com/benoitc/inet_cidr.git 1.0.2
+# used by kz_network_utils
 
 dep_jesse = git https://github.com/2600hz/jesse 1.5-rc13
-##dep_jesse = git https://github.com/for-GET/jesse 1.5.0-rc2
-
-dep_lager = git https://github.com/erlang-lager/lager 3.5.2
-dep_trie = git https://github.com/okeuday/trie v1.5.4
-dep_fs_event = git https://github.com/jamhed/fs_event 783400da08c2b55c295dbec81df0d926960c0346
-dep_fs_sync = git https://github.com/jamhed/fs_sync 2cf85cf5861221128f020c453604d509fd37cd53
-dep_inet_cidr = git https://github.com/icehess/inet_cidr.git
-### PR opened upstream ###
-dep_erlang_localtime = git https://github.com/lazedo/erlang_localtime 0bb26016380cd7df5d30aa0ef284ae252b5bae31
+# used by kazoo_schemas primarily
+
+dep_jiffy = git https://github.com/2600hz/jiffy master  ## utf8 decode
+# includes changes from lazedo/utf8
+# used by kz_json, nklib, jesse, lager, maybe couchbeam if compiled
 
-### need to update upstream ###
-dep_hep = git https://github.com/lazedo/hep 1.5.4
-dep_ecsv = git https://github.com/lazedo/ecsv ecsv-1
+dep_lager = git https://github.com/erlang-lager/lager 3.6.10
+# used everywhere
 
-### for scripts/dev-start-*.sh
-dep_reloader = git https://github.com/lazedo/reloader v0.1
+dep_lager_syslog = git https://github.com/erlang-lager/lager_syslog 3.0.3
 
-### build
-dep_parse_trans = git https://github.com/lazedo/parse_trans
+dep_meck = git https://github.com/eproxus/meck 0.8.13
+# used in tests for kazoo_voicemail, crossbar, teletype, and other deps
 
-dep_horse = git https://github.com/ninenines/horse
+# dep_meta = git https://github.com/efcasado/meta 0.1.3
+# appears unused
```

Comment:
this is used

Reply:
Where? How? `meta:` isn't found anywhere in the code?